open-banking-accelerator/components/com.wso2.openbanking.accelerator.identity/src/main/java/com/wso2/openbanking/accelerator/identity/push/auth/extension/request/validator/util/PushAuthRequestValidatorUtils.java

```diff
@@ -217,6 +217,13 @@ public static void validateScope(JSONObject requestBodyJson) throws PushAuthRequ
             List<String> requestedScopes = Arrays.asList(requestBodyJson.getAsString(PushAuthRequestConstants.SCOPE)
                     .split("\\s+"));
 
+            if (!requestedScopes.contains(OAuthConstants.Scope.OPENID)) {
```

Comment:
openid scope is mandated in CDS spec right? Since we are doing a accelerator level change, don't we need a config? otherwise this may not be backward compatible right?

Reply:
fixed with a1e6c40a1dbefb8af184788bb0cef6d80b5da367